db/schema.rb

```diff
@@ -141,20 +141,21 @@
   end
 
   create_table "project_contents", force: :cascade do |t|
-    t.integer  "project_id",        index: {name: "index_project_contents_on_project_id"}
+    t.integer  "project_id",           index: {name: "index_project_contents_on_project_id"}
     t.string   "language"
-    t.string   "title",             default: ""
-    t.text     "description",       default: ""
+    t.string   "title",                default: ""
+    t.text     "description",          default: ""
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.text     "introduction",      default: ""
-    t.text     "science_case",      default: ""
+    t.text     "introduction",         default: ""
+    t.text     "science_case",         default: ""
     t.json     "team_members"
     t.json     "guide"
-    t.text     "faq",               default: ""
-    t.text     "result",            default: ""
-    t.text     "education_content", default: ""
-    t.jsonb    "url_labels",        default: {}
+    t.text     "faq",                  default: ""
+    t.text     "result",               default: ""
+    t.text     "education_content",    default: ""
+    t.jsonb    "url_labels",           default: {}
+    t.text     "workflow_description", default: "", null: false
```

Comment:
No probs, we should eventually migrate these cols to default and set null: false.